open-sphere-plugins/wms/src/main/java/io/opensphere/wms/WMSTransformer.java

```diff
@@ -272,63 +272,66 @@ public void timespansChanged(Collection<? extends TimeSpan> addedSpans, Collecti
     @Override
     public void activeTimeChanged(TimeSpan active)
     {
-        fadeTiles(active);
+        // Intentionally disabling WMS tile fading since it causes tiles to
+        // turn transparent if the active span is less than a day. Code kept in
+        // case improvements can be made upon the fading algorithm.
+//        fadeTiles(active);

```

Comment:
Can we just not override this method and it will work the same as commenting it all out?